Tests/Runtime/CinemachineFixtureBase.cs

```diff
@@ -7,12 +7,12 @@ namespace Tests.Runtime
 {
     public class CinemachineFixtureBase
     {
-        private List<GameObject> m_GameObjectsToDestroy = new List<GameObject>();
+        private readonly List<GameObject> _gameObjectsToDestroy = new List<GameObject>();
```

Comment:
Why did you remove the "m"?  Have the unity coding standards changed since I last looked?
We're trying to slowly refactor to follow this:
https://github.com/Unity-Technologies/dots/blob/master/CONVENTIONS.md

(I know it says DOTS, but it extends Unity's global coding conventions, which are here: https://github.cds.internal.unity3d.com/raw/unity/com.unity.coding/master/UnityCoding/Packages/com.unity.coding/Coding~/Conventions/CSharpReference.cs?token=AAAAHFCSSOEUQ6IIT2HISZDA7BMJY )

Reply:
I'm out of date, I thought we were following Microsoft's conventions. I'll revert the renaming.